src/views/router_views/start/start.njk

```diff
@@ -3,81 +3,31 @@
-        {{ i18n.start_service_new_pscs_end }}
-    </p>
     <p class="govuk-body">{{ i18n.start_service_restriction_description }}</p>
-
-    {% set startFindDetailsText %}
-    <h3 class="govuk-heading-s">{{ i18n.start_find_details_psc_heading }}</h3>
-    <p class="govuk-body">{{ i18n.start_find_details_psc_list }}</p>
-    <ul class="govuk-list govuk-list--bullet">
-        <li>{{ i18n.start_find_details_psc_list_item1 }}</li>
-        <li>{{ i18n.start_find_details_psc_list_item2 }}</li>
-        <li>{{ i18n.start_find_details_psc_list_item3 }}</li>
-    </ul>
-
-    <h3 class="govuk-heading-s">{{ i18n.start_find_details_rle_heading }}</h3>
-    <p class="govuk-body">{{ i18n.start_find_details_rle_description }}</p>
-    <p class="govuk-body">{{ i18n.start_find_details_rle_list }}</p>
-    <ul class="govuk-list govuk-list--bullet">
-        <li>{{ i18n.start_find_details_rle_list_item1 }}</li>
-        <li>{{ i18n.start_find_details_rle_list_item2 }}</li>
-    </ul>
-    {% endset %}
-
-    {{ govukDetails({
-            summaryText: i18n.start_find_details_main_link,
-            html: startFindDetailsText,
-            attributes: {
-                "data-event-id": "find-more-about-dropdown"
-            }
-        }) }}
-
-    <h2 class="govuk-heading-m">{{ i18n.start_when_to_verify_heading }}</h2>
-    <p class="govuk-body">{{ i18n.start_when_to_verify_description }}{{ idvImplementationDate }}</p>
-    <p class="govuk-body">{{ i18n.start_when_to_verify_not_early }}</p>
-
-    <h3 class="govuk-heading-s">{{ i18n.start_when_individual_heading }}</h3>
-    <p class="govuk-body">
-        {{ i18n.start_when_individual_before_date_1 }}{{ idvImplementationDate }}{{ i18n.start_when_individual_before_date_2 }}
-    </p>
-    <p class="govuk-body">
-        {{ i18n.start_when_individual_after_date_1 }}{{ idvImplementationDate }}{{ i18n.start_when_individual_after_date_2 }}
-    </p>
-
-    <h3 class="govuk-heading-s">{{ i18n.start_when_rle_heading }}</h3>
-    <p class="govuk-body">
-        {{ i18n.start_when_rle_before_date_1 }}{{ idvImplementationDate }}{{ i18n.start_when_rle_before_date_2 }}
-    </p>
-    <p class="govuk-body">
-        {{ i18n.start_when_rle_after_date_1 }}{{ idvImplementationDate }}{{ i18n.start_when_rle_after_date_2 }}
-    </p>
-
-    {{ govukInsetText({
-        text: i18n.start_when_inset
+    
+    <h2 class="govuk-heading-m">{{ i18n.start_when_to_verify_heading }}</h1>
+     <p class="govuk-body">{{ i18n.start_when_to_verify_description1 }}</p>
+     <p class="govuk-body">{{ i18n.start_when_to_verify_description2 }}</p>
+     <p class="govuk-body">{{ i18n.start_when_to_verify_description3 }}</p>
+
+     {{ govukWarningText({
+        text: i18n.start_warning_text
     }) }}
 
-    <h2 class="govuk-heading-m">{{ i18n.start_before_heading }}</h2>
+        <h2 class="govuk-heading-m">{{ i18n.start_before_heading }}</h2>
     <p class="govuk-body">{{ i18n.start_before_verify_description }}
```

Comment:
Please update lines 31 to 35 as follows:

    <p class="govuk-body">{{ i18n.start_before_verify_link }}
        <a href="{{ [chsUrl, verifyIdentityLink] | join  }}" title="{{ i18n.start_verify_link_text }}" class="govuk-link" data-event-id="before-verify-link">
            {{ i18n.start_verify_link_text }}
        </a>
        {{i18n.start_after_verify_link}}</p>